huaweicloud/services/acceptance/acceptance.go

```diff
@@ -565,6 +569,36 @@ func TestAccPreCheckChargingMode(t *testing.T) {
 	}
 }
 
+// lintignore:AT003
+func TestAccPreCheckEVSQoSRegion(t *testing.T) {
+	if HW_EVS_QOS_REGION != "cn-south-1" {
```

Comment:
What if I want to run test cases in `cn-north-4` region?
I suggest you only check whether availability is written here, and add necessary descriptions to `PreCheck`.